puppet-lint-empty_trailing_lines.gemspec

```diff
@@ -0,0 +1,26 @@
+Gem::Specification.new do |spec|
+  spec.name        = 'puppet-lint-empty_trailing_lines'
+  spec.version     = '0.0.1'
+  spec.homepage    = 'https://github.com/ghoneycutt/puppet-lint-empty_trailing_lines'
+  spec.license     = 'Apache-2.0'
+  spec.author      = 'Garret Honeycutt'
+  spec.email       = 'gh@tailoredautomation.io'
+  spec.files       = Dir[
+    'README.md',
+    'LICENSE',
+    'lib/**/*',
+    'spec/**/*',
+  ]
+  spec.test_files  = Dir['spec/**/*']
+  spec.summary     = 'puppet-lint empty_trailing_line check'
+  spec.description = <<-EOF
+    Extends puppet-lint to ensure that your manifest files do not end with extra empty lines
+  EOF
+
+  spec.add_dependency             'puppet-lint', '>= 1.0', '< 3.0'
```

Comment:
We should probably just support puppet-lint v2

Reply:
Updated